conf/index-settings.json

```diff
@@ -43,7 +47,11 @@
 					}
 				}, 
 				"id": {
-					"index": "not_analyzed", 
+					"analyzer": "id",
+					"type": "string"
+				},
+				"isil": {
+					"analyzer": "id",
 					"type": "string"
 				}
```

Comment:
They are both set to `analyzer: id`, the `not_analyzed` for the id field was removed.

The reason is to have the same behavior for both (to work both with upper- and lowercase).